srcpkgs/nwg-bar/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'nwg-bar'
+pkgname=nwg-bar
+version=0.1.6
+revision=1
+build_style=go
+go_import_path=github.com/nwg-piotr/nwg-bar
+hostmakedepends="pkg-config"
+makedepends="python3-gobject-devel gtk+3-devel gtk-layer-shell-devel"
+depends="gtk-layer-shell gtk+3 zlib pango libharfbuzz atk cairo gdk-pixbuf glib gtk-layer-shell fontconfig freetype glibc"
```

Comment:
Those dependencies are all automatically detected.
https://github.com/void-linux/void-packages/actions/runs/15530005489/job/43716751969?pr=55669#step:7:186

Reply:
So I shoudn't use depends variable in templates at all, because will be detected automatically?

Reply:
See the difference from the first log:
```
 => nwg-bar-0.1.6_1: running pre-pkg hook: 999-collected-rdeps ...
   gtk-layer-shell>=0 gtk+3>=0 zlib>=0 pango>=0 libharfbuzz>=0 atk>=0 cairo>=0 gdk-pixbuf>=0 glib>=0 gtk-layer-shell>=0 fontconfig>=0 freetype>=0 glibc>=0 gtk+3>=3.0.0_1 zlib>=1.2.3_1 pango>=1.24.0_1 libharfbuzz>=5.1.0_1 atk>=1.26.0_1 cairo>=1.8.6_1 gdk-pixbuf>=2.22.0_1 glib>=2.80.0_1 gtk-layer-shell>=0.1.0_1 fontconfig>=2.6.0_1 freetype>=2.12.1_1 glibc>=2.41_1 
```

vs the new log:

```
=> nwg-bar-0.1.6_1: running pre-pkg hook: 999-collected-rdeps ...
   gtk+3>=3.0.0_1 zlib>=1.2.3_1 pango>=1.24.0_1 libharfbuzz>=5.1.0_1 atk>=1.26.0_1 cairo>=1.8.6_1 gdk-pixbuf>=2.22.0_1 glib>=2.80.0_1 gtk-layer-shell>=0.1.0_1 fontconfig>=2.6.0_1 freetype>=2.12.1_1 glibc>=2.41_1 
```